web-components/src/components/modal/BridgeModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { BridgeForm, BridgeProps } from '../form/BridgeForm';
+import { ToucanIcon } from '../icons/ToucanIcon';
+import { RegenModalProps } from '../modal';
+import { FormModalTemplate } from './FormModalTemplate';
+
+interface BridgeModalProps extends RegenModalProps, BridgeProps {}
+
+export const BRIDGE_TITLE = 'Bridge Ecocredits to Polygon';
+const BRIDGE_SUBTITLE =
+  'This service allows you to bridge NCT-eligible ecocredits to Polygon in the form of TCO2s.';
+
+const BridgeModal: React.FC<BridgeModalProps> = ({
```

Comment:
i could be misrembering this but are we trying to move away from using `React.FC`, i'm vaguely feeling like i heard this in a prior code review..

Reply:
good call. Changed both instances